packages/shared/ReactTypeOfSideEffect.js

```diff
@@ -9,16 +9,31 @@
 
 export type TypeOfSideEffect = number;
 
-// Don't change these two values:
-export const NoEffect = 0b00000000;
-export const PerformedWork = 0b00000001;
+// Don't change these two values. They're used by React Dev Tools.
+export const NoEffect = /*              */ 0b000000000000;
+export const PerformedWork = /*         */ 0b000000000001;
 
 // You can change the rest (and add more).
-export const Placement = 0b00000010;
-export const Update = 0b00000100;
-export const PlacementAndUpdate = 0b00000110;
-export const Deletion = 0b00001000;
-export const ContentReset = 0b00010000;
-export const Callback = 0b00100000;
-export const Err = 0b01000000;
-export const Ref = 0b10000000;
+export const Placement = /*             */ 0b000000000010;
+export const Update = /*                */ 0b000000000100;
+export const PlacementAndUpdate = /*    */ 0b000000000110;
+export const Deletion = /*              */ 0b000000001000;
+export const ContentReset = /*          */ 0b000000010000;
+export const Callback = /*              */ 0b000000100000;
+export const DidCapture = /*            */ 0b000001000000;
+export const Ref = /*                   */ 0b000010000000;
+export const ErrLog = /*                */ 0b000100000000;
+
+// Union of all host effects
+export const HostEffectMask = /*        */ 0b000111111111;
+
+// This indicates we exited the fiber without completing it.
+export const Incomplete = /*            */ 0b001000000000;
+
+// Bits greater than Incomplete are used to identify fibers that threw. These
+// are like algebraic effects. These bits are mutually exclusive.
+export const Combined = /*              */ 0b010000000000;
```

Comment:
Can you explain why this is called "combined"? What is being combined here? Isn't the first effect that we "catch" the only one? Or do we attempt to render siblings as deep as we can and then "wait on all"?

Reply:
I think I'll just get rid of this, but the idea was that you could dispatch multiple errors at the same time. It's only being used for unhandled errors at the root. Don't think it's necessary.